.github/workflows/terraform-plan-apply.yml

```diff
@@ -202,15 +220,15 @@ jobs:
 
               var_name=${line_array[0]}
               json_path=${line_array[1]}
-              json_value=$(jq -r ${json_path} ${{ matrix.definition_path }})
+              json_value=$(yq eval ${json_path} ${{ matrix.definition_path }})
```

Comment:
can you change from json to something else?
property_path / property_value?

Reply:
Done